pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -34,6 +34,8 @@ const (
 	PodCIDRSelectorName = "PodCIDR"
 	// CiliumLoadBalancerIPPoolSelectorName defines the name for a selector matching CiliumLoadBalancerIPPool resources.
 	CiliumLoadBalancerIPPoolSelectorName = "CiliumLoadBalancerIPPool"
+	// CiliumPodIPPoolSelectorName defines the name for a selector matching CiliumPodIPPool resources.
+	CiliumPodIPPoolSelectorName = "CiliumPodIPPool"
```

Comment:
Do we need this constant? We do have "CiliumPodIPPool" defined [here](https://github.com/cilium/cilium/blob/main/pkg/k8s/apis/cilium.io/v2alpha1/register.go#L80 )

Reply:
@harsimran-pabla thanks for the pointer. I was following the existing constant pattern but I'll update to use your ref.

Reply:
@harsimran-pabla commit 51738e1 keeps the `CiliumPodIPPoolSelectorName` const but now refers to [the associated Kind](https://github.com/cilium/cilium/blob/main/pkg/k8s/apis/cilium.io/v2alpha1/register.go#L80).